internal/pkg/cli/task_run.go

```diff
@@ -136,7 +136,7 @@ func newTaskRunOpts(vars runTaskVars) (*runTaskOpts, error) {
 	opts.configureRuntimeOpts = func() error {
 		opts.runner = opts.configureRunner()
 		opts.deployer = cloudformation.New(opts.sess)
-		opts.defaultClusterGetter = ecs.New(opts.sess)
+		opts.defaultClusterGetter = awsecs.New(opts.sess)
```

Comment:
it'd be nice if we can move this functionality too to the new `ecs` pkg to remove the dependency on the aws sdk

Reply:
Actually I take that back, I think what you have makes sense, since the goal of `awsecs` is to be copilot agnostic whereas this package is only to fetch Copilot related ECS resources